packages/cozy-client/types/models/diskUsage.d.ts

```diff
@@ -0,0 +1,29 @@
+export function makeDiskInfos(usage: number | string, quota?: number | string): DiskInfos;
+export type DiskInfosRaw = {
+    /**
+     * - Space used in GB
+     */
+    diskQuota: number;
+    /**
+     * -  Maximum space available in GB
+     */
+    diskUsage: number;
+    /**
+     * - Usage percent of the disk
+     */
+    percentUsage: number;
+};
+export type DiskInfos = {
+    /**
+     * - Space used in GB rounded
+     */
+    humanDiskQuota: string;
+    /**
+     * - Maximum space available in GB rounded
+     */
+    humanDiskUsage: string;
+    /**
+     * - Usage percent of the disk rounded
+     */
+    percentUsage: string;
+};
```

Comment:
To remove?

Reply:
Why? 🤔 I'm currently using it in drive show my banner. Do you think it's the responsibility of each app to make this calculation according to their needs?

Reply:
Fixed 🔧